app/src/main/AndroidManifest.xml

```diff
@@ -43,7 +39,13 @@
         <activity
             android:name=".OutfitCreationActivity"
             android:exported="true"
-            android:theme="@style/Theme.Design.Light.NoActionBar"></activity>
+            android:theme="@style/Theme.Design.Light.NoActionBar">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
```

Comment:
launch on outfit creation for testing - should be changed based on which page is being testes